gtsfm/multi_view_optimizer.py

```diff
@@ -73,26 +72,23 @@ def create_computation_graph(
             List of GtsfmMetricGroups from different modules, wrapped up as Delayed.
         """
 
-        if self._run_view_graph_estimator and self.view_graph_estimator is not None:
-            (
-                viewgraph_i2Ri1_graph,
-                viewgraph_i2Ui1_graph,
-                viewgraph_v_corr_idxs_graph,
-                viewgraph_two_view_reports_graph,
-                viewgraph_estimation_metrics,
-            ) = self.view_graph_estimator.create_computation_graph(
-                i2Ri1_graph, i2Ui1_graph, all_intrinsics, v_corr_idxs_graph, keypoints_graph, two_view_reports_dict
-            )
-        else:
-            viewgraph_i2Ri1_graph = dask.delayed(i2Ri1_graph)
-            viewgraph_i2Ui1_graph = dask.delayed(i2Ui1_graph)
-            viewgraph_v_corr_idxs_graph = dask.delayed(v_corr_idxs_graph)
-            viewgraph_two_view_reports_graph = dask.delayed(two_view_reports_dict)
-            viewgraph_estimation_metrics = dask.delayed(GtsfmMetricsGroup("view_graph_estimation_metrics", []))
-
-        # prune the graph to a single connected component.
-        pruned_i2Ri1_graph, pruned_i2Ui1_graph = dask.delayed(graph_utils.prune_to_largest_connected_component, nout=2)(
-            viewgraph_i2Ri1_graph, viewgraph_i2Ui1_graph, relative_pose_priors
+        if self._run_view_graph_estimator is None:
```

Comment:
we should remove this flag altogether:
- in initializer view_graph_estimator argument should be non-optional. 
- all the configs that dont use view_graph_estimator should be updated to use ViewGraphWithoutFiltering.

Reply:
addressed [78c7180](https://github.com/borglab/gtsfm/pull/569/commits/78c71804669811a00ae0053c59d054b37f39cea0) and [ddae6be](https://github.com/borglab/gtsfm/pull/569/commits/ddae6be1906674515722b8ad6a672428dd66bc20)